services/dr-manhattan/louvaine.py

```diff
@@ -70,12 +72,12 @@ def get_text_sum(self, cluster, r_o):
                     else:
                         r_o["campaigns"]["ids"][cam] = 1
 
-            if r is not None:
-                for res in r.json():
+            if entities:
+                for res in entities:
```

Comment:
Need to test this syntax with QCR kafka - there may be version specific issues with this.